pep-0676.rst

```diff
@@ -74,38 +77,42 @@ There are several requests for additional features in reading PEPs, such as:
 * support for SVG images [3]_
 * typographic quotation marks [4]_
 * additional footer information [5]_
-* intersphinx functionality [6]_
+* Intersphinx functionality [6]_
 
 These are "easy wins" from this proposal, and would serve to improve the
 quality-of-life for consumers of PEPs (including reviewers and writers). For
-example, the reference implementation no longer requires a scheduled render
+example, the new reference implementation no longer requires a scheduled render
 process to run, instead initiating rendering on every commit to the
 `python/peps`_ repository.
 
-Equally, there are a small number of broken items, for example list styles not
-being respected or support for updating images being challenging with the
-system [7]_. These would be solved by default in the proposal.
+Equally, there are a number of broken items currently; for example, list styles
+are not respected, and support for updating images is challenging with the
+current system [7]_. These are solved by the reference implementation of
+this proposal.
```

Comment:
I like the simplification. However:

* It doesn't make much sense to use, and many style guides explicitly prohibit, an unordered list for only two items (three is typically the minimum), without strong justification. It would be ideal to add a third item if you have one, but otherwise the listification would best be reverted.

* Furthermore, both items in the list reference the same footnote, which is redundant and doesn't really make sense, so that should either go next to "several small issues", or perhaps better, convert the footnote to individual direct inline links.

* Finally, some of the wording changes that were avoided were important to avoid unnecessarily minimizing and trivializing the fixes, making a more compelling case for the PEP. If you don't like the particular words chosen, I can certainly suggest others that would still avoid this issue.

Reply:
Hmm,

* I think two bullet points are just fine, especially if it helps lay out the information more clearly and improves readability (people don't read screens, they scan) 

* I'd be interested to see what style guides prohibit two, the [first I found](https://www.stylemanual.gov.au/structuring-content/lists) talking about bullet points has several with just two! And [Google's style guide](https://developers.google.com/style/lists) is usually well thought out, they also use and don't prohibit two.